packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -503,6 +504,86 @@ class ContainerLayer extends Layer {
     return child == equals;
   }
 
+  PictureLayer _highlightConflictingLayer(PhysicalModelLayer child) {
+    final ui.PictureRecorder recorder = ui.PictureRecorder();
+    final Canvas canvas = Canvas(recorder);
+    canvas.drawPath(
+      child.clipPath,
+      Paint()
+        ..color = const Color(0xFFAA0000)
+        ..style = PaintingStyle.stroke
+        ..strokeWidth = child.elevation + 10.0,
```

Comment:
10.0 may be violating https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-mysterious-and-magical-numbers-that-lack-a-clear-derivation

Reply:
I'm not sure of a better way to do this.  Any ideas?

Reply:
Style guide says you could add a comment explaining the 10.0

Reply:
Done